packages/local_auth/local_auth/README.md

```diff
@@ -252,6 +252,37 @@ types (such as face scanning) and you want to support SDKs lower than Q,
 _do not_ call `getAvailableBiometrics`. Simply call `authenticate` with `biometricOnly: true`.
 This will return an error if there was no hardware available.
 
+#### Android theme
+
+On Android, ensure that the `LaunchTheme` of your app has a parent style with the `Theme.AppCompat.Light` theme; otherwise the app may crash. To do that go to `android > app > src > main > res > values > styles.xml` and look for the style with name `LaunchTheme` (Notice that `LaunchTheme` must be referenced in the `AndroidManifest.xml` file). Then, change the parent style with the new value and you should have set up something like this:
```

Comment:
I don't understand the parenthetical; why does someone need to "notice" something in order to make this change, particularly something in another file?

Reply:
In `AndroidManifest.xml` you specify the name of the style you have defined in the `styles.xml`, therefore if you don't notice that you have referenced the correct name the theme in `styles.xml` won't apply.

I updated this part telling the reader that this reference makes possible the application of the defined theme in `styles.xml`:

(Notice that `LaunchTheme` must be referenced in the `AndroidManifest.xml` file to apply the changes made in `styles.xml`)

Reply:
> if you don't notice that you have referenced the correct name the theme in `styles.xml` won't apply

Whether or not someone *notices* that its' there is irrelevant to whether it works, only whether it *is* there. So this is telling someone to do something that doesn't help ("notice" something) if it is there, and also doesn't tell them what to do if it's not. The instructions here for `styles.xml` seem to assume that the reader isn't familiar with any of this and thus needs extremely detailed instructions; the instructions for `AndroidManifest.xml` should equally clear and detailed, assuming the same level of reader knowledge.

Reply:
I understand. Well, due to Flutter creates the `styles.xml` always with a `LaunchTheme` style name and it is associated to the `AndroidManifest.xml` by default, I will remove the phrase between parentheses (...).